src/App.js

```diff
@@ -1,25 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+// eslint-disable-next-line import/extensions
+import Calculator from './components/Calculator.js';
```

Comment:
- [ ] Your app produces an error the cause is the naming of Calculator, kindly consider fixing this issue so your app is working as intended

![Screenshot_20220713_203552](https://user-images.githubusercontent.com/25227022/178817732-c3d92a09-4d33-4a84-8b7f-f92ef700a0a4.png)